### ISSUE TITLE
Val and Test images all have the same orientation but actual test/val has variation of orientation

### ISSUE THREAD

**Issue by georgeblu1** [ultralytics/ultralytics#10310]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi! For some reason, my test and val dataset after running the model.val, all of them have the same orientation. I am not sure if there's an ordering in terms of data involved or non of my other data were loaded into it for validation. Additionally, is there possibility to enable shuffling for output images during metrics validation?

```
model = YOLO(pt_model_path)


test_metrics = model.val(
    data="custom-train-seg-local.yaml",
    split="test",
    project="artifacts/runs",
    name="_detection_test",
    device = "mps",
    iou= 0.6,
)
```

The model is trained on images with multiple orientation but based on the run output directory, for val and test, all of them is showing the same orientation only.

Unable to submit images due to confidentiality

**Comment by glenn-jocher** [comment]
Hi there! You're welcome. 😊 Indeed, the `.val` method uses the dataset as is, without an inherent shuffling option. If you're interested in shuffling your data for validation purposes, consider shuffling your dataset prior to validation. This could be achieved within your dataset preparation phase. For example:

```python
from sklearn.utils import shuffle
X, y = shuffle(X, y, random_state=42)
```

Just ensure `X` and `y` represent your dataset and labels respectively, and adjust `random_state` as needed for reproducibility. This is just a guideline; adapt it to fit your specific setup. Happy coding! 🚀
